pg_back

```diff
@@ -210,33 +215,37 @@ if [ $? != 0 ]; then
     info "could not check for replication control functions"
 fi
 
-# Pause replay if dumping from a slave
+# Pause replay if dumping from a standby
 if [ "${PG_HASPAUSE}" = "1" ]; then
     info "pausing replication replay"
 
-    # Wait for exclusive locks to get released on the slave before
+    # Wait for exclusive locks to get released on the standby before
     # pausing the replication replay
     typeset -i PGBK_EXCLLOCKS;
     PGBK_EXCLLOCKS=1;
-    while [ $PGBK_EXCLLOCKS -gt 0  ]
+    PGBK_PAUSE=10
+    while [ $PGBK_EXCLLOCKS -gt 0 ]
     do
-        ${PGBK_BIN}psql -X $OPTS -At -c "SELECT pg_${xlog_or_wal}_replay_pause() where pg_is_in_recovery();" $PGBK_CONNDB
-        if [ $? != 0 ]; then
-            die "could not pause replication replay"
-        fi
         PGBK_EXCLLOCKS=`${PGBK_BIN}psql -X $OPTS -At -c "SELECT count(*) FROM pg_locks WHERE mode = 'AccessExclusiveLock';" $PGBK_CONNDB`
         if [ $? != 0 ]; then
             die "could not get lock information"
         fi
+        
         if [ $PGBK_EXCLLOCKS -gt 0 ]; then
-            ${PGBK_BIN}psql -X $OPTS -At -c "SELECT pg_${xlog_or_wal}_replay_resume();" $PGBK_CONNDB
+            if [ $PGBK_STANDBY_PAUSE_TIMEOUT -le 0 ]; then
+                die "attempt to pause replication exceeded timeout"
+            fi
+
+            echo "The standby database has exclusive locks (vacuum full, truncate or other locking command) running on primary"
+            echo "Resuming replication for ${PGBK_PAUSE}s"
```

Comment:
I guess we could use warn() and info() here to print messages correctly. Could you please make to change?

Thank you